index.html

```diff
@@ -9,31 +9,458 @@
+                <path d="M-56.7443 16.5674L15.3431 43.4153L0.864263 48.5292L-35.1539 56.8648L-56.7443 16.5674Z"
+                    fill="white" stroke="#36B37E" stroke-width="0.633632" stroke-miterlimit="10"
+                    stroke-linejoin="round" />
+                <path d="M15.3431 43.4156L47.6264 0.612305L26.4964 45.2055L15.3431 43.4156Z" fill="white"
+                    stroke="#36B37E" stroke-width="0.633632" stroke-miterlimit="10" stroke-linejoin="round" />
+                <path d="M-14.7914 32.2166L0.0967395 15.4941L2.09206 16.5681L-0.0567445 37.6884L-14.7914 32.2166Z"
+                    fill="#36B37E" />
+                <path d="M2.09206 16.5674L20.9709 35.949L15.343 43.4153L-0.0567474 37.6878L2.09206 16.5674Z"
+                    fill="white" stroke="#36B37E" stroke-width="0.633632" stroke-miterlimit="10"
+                    stroke-linejoin="round" />
+                <path d="M26.4964 45.2049L0.864197 48.5289L15.3431 43.415L26.4964 45.2049Z" fill="white"
+                    stroke="#36B37E" stroke-width="0.633632" stroke-miterlimit="10" stroke-linejoin="round" />
+                <mask id="mask0_73_258" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="-58" y="16" width="74"
+                    height="41">
+                    <path d="M-56.8336 16.7588L15.2077 43.3089L0.941376 48.2228L-35.0768 56.5584L-56.8336 16.7588Z"
+                        fill="white" stroke="#36B37E" stroke-width="0.633632" stroke-miterlimit="10"
+                        stroke-linejoin="round" />
+                </mask>
+                <g mask="url(#mask0_73_258)">
+                    <path
+                        d="M-13.4611 44.4895L-21.2889 44.7452L-24.9214 38.4551L-32.4423 38.7619L-28.9121 45.0009L-36.3306 45.2054L-32.9027 51.4955L-25.3307 51.3421L-17.6052 51.1375L-13.8192 57.7345L-5.73563 57.6322L-1.74498 64.3825H6.64561L2.55264 57.5299L-1.48917 50.8307L-9.67511 50.9841L-13.4611 44.4895Z"
+                        fill="#C5E9DA" />
+                    <path
+                        d="M2.75744 43.9268L-5.47966 44.2337L-1.48902 50.8306L6.85041 50.626L15.4457 50.4215L11.1992 43.6711L7.05506 37.0742L-1.23321 37.4322L2.75744 43.9268Z"
+                        fill="#36B37E" />
+                    <path d="M-0.977467 24.2383L-9.00992 24.7497L-5.17276 30.9886L3.01318 30.5795L-0.977467 24.2383Z"
+                        fill="#36B37E" />
+                    <path d="M15.5991 36.6649L11.3526 30.1191L3.01319 30.5794L7.05499 37.074L15.5991 36.6649Z"
+                        fill="#36B37E" />
+                </g>
+            </svg>
         </div>
-    </article>
+        <div>
+            <h1>About Me</h1>
+            <p>I can help you build a product, feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <button class="btn-green">Get My Resume</button>
+        </div>
```

Comment:
- [ ] Consider wrapping the `About me` section in a `section` tag  instead of a `div` to improve the [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) structure of your HTML.

Reply:
`About me` is already nested in `<section>`. Wrapping again will break down the symmetry.